src/librustdoc/core.rs

```diff
@@ -317,6 +318,12 @@ pub(crate) fn create_config(
                 }
                 (rustc_interface::DEFAULT_QUERY_PROVIDERS.type_of)(tcx, def_id)
             };
+            providers.codegen_fn_attrs = |_, did| {
+                assert!(did.is_local());
+                CodegenFnAttrs::new()
```

Comment:
Is this right? Won't it end up in us failing to include the target features in the generated docs?

cc @Amanieu , I'm not sure if this has tests.